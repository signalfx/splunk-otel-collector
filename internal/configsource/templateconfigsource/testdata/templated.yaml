config_sources:
  template:

component_0: |
  $template: ./testdata/component_template
  glob_pattern: /var/**/*.log
  format: json

component_1: $template:./testdata/no_params_template

component_2: |
  $template: ./testdata/yaml_template
  k0: 42
  cfg:
    k0: a_string
    k1: true
