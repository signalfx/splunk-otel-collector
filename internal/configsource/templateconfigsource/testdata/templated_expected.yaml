component_0:
  logs_path: /var/**/*.log
  log_format: json

component_1:
  bool_field: true

component_2:
  int_field: 42
  map:
    str_field: a_string
    bool_field: true
